tests/test_job.py

```diff
@@ -1286,6 +1286,7 @@ def test_job_wait_for_close(hq_env: HqEnv):
     time.sleep(1)
     assert p.poll() is None
     hq_env.command(["job", "close", "last"])
+    wait_for_job_state(hq_env, 1, "FINISHED")
     time.sleep(0.3)
     r = p.poll()
```

Comment:
Let's remove the `sleep(0.3)` call and use `p.wait(5)` here instead, it should be less flaky.